pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -520,7 +520,7 @@ stdenv.mkDerivation {
     ''
 
     # TODO: categorize these and figure out a better place for them
-    + optionalString hostPlatform.isCygwin ''
+    + optionalString (hostPlatform.isCygwin || targetPlatform.isMSVC) ''
```

Comment:
ah good catch! It did not occur to me that this isn't a UEFI specific limitation but a broader Windows limitation.

Not something we need to resolve for this PR to go through but: [should this conditional include `MinGW`](https://bugs.webkit.org/show_bug.cgi?id=163949)?

Reply:
I don't think so, because I can cross-build packages for mingw just fine at the moment.

Reply:
Oh, I see the linked bug report mentions it just doesn't do anything on MinGW.  So in that case I guess it should just be `targetPlatform.isWindows`.